src/api/api.ts

```diff
@@ -121,6 +122,7 @@ const errorHandler: ErrorRequestHandler = (err, req, res, _next) => {
   res.status(500).json({
     message:
       'Something unexpected went wrong. If the problem persists, please contact support with details about what you were trying to do.',
+    errorHash: uuid(), // TODO: should come from winston request context
```

Comment:
Can this be populated or is it dependent on something coming up? Can you throw up a quick ticket about what needs to be done?

Reply:
Yup, this is dependent on the application side request logging. I see there is a ticket [here](https://atlassian.thetradedesk.com/jira/browse/UID2-873) for node application logging, and anthony has a PR on winston [here](https://github.com/IABTechLab/uid2-self-serve-portal/pull/23). 

I'll check with @amay-ttd and get it implemented.